contracts/TwabRewards.sol

```diff
@@ -256,7 +256,7 @@ contract TwabRewards is ITwabRewards {
             (_promotion.epochDuration * _promotion.numberOfEpochs);
 
         require(
-            _promotionEndTimestamp > 0 && _promotionEndTimestamp >= block.timestamp,
+            _promotionEndTimestamp >= block.timestamp,
```

Comment:
Ah!  We covered this in the last PR.  Should be `>`

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/29/commits/bdab963b5f3e321b412d8f2862baf328fc4975ee